specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/search.json

```diff
@@ -777,6 +828,266 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Search/searchServices/{searchServiceName}/privateEndpointConnections": {
+      "get": {
+        "tags": [
+          "PrivateEndpointConnections"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "operationId": "PrivateEndpointConnections_ListByService",
+        "x-ms-examples": {
+          "ListPrivateEndpointConnectionsByService": {
+            "$ref": "./examples/ListPrivateEndpointConnectionsByService.json"
+          }
+        },
+        "description": "Gets a list of all private endpoint connections in the given service.",
+        "externalDocs": {
+          "url": "https://aka.ms/search-manage"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ClientRequestIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SearchServiceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the list of all private endpoint connections for the given service.",
+            "schema": {
+              "$ref": "#/definitions/PrivateEndpointConnectionListResult"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred during the operation.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Search/searchServices/{searchServiceName}/sharedPrivateLinkResources/{sharedPrivateLinkResourceName}": {
+      "put": {
+        "tags": [
+          "SharedPrivateLinkResources"
+        ],
+        "operationId": "SharedPrivateLinkResources_Update",
```

Comment:
This needs to be "_CreateOrUpdate", and please modify the description accordingly.

Reply:
This was copied from the one for privateEndpointConnections, so I was just trying to be consistent.  Should I update that one to be CreateOrUpdate too?

Reply:
I didn't update the other one, but I did update it here.

Reply:
Right, it's accurate for the "privateEndpointConnections", because the "Create" private endpoint into Azure Search goes to Network RP and not ours... only calls to "Update" come to us...

Whereas for shared private link resources, we handle both create and update